examples/session/session-reservation.py

```diff
@@ -167,6 +167,10 @@ def ThrowOnError (vi, error_code):
     print(f'All sessions have been closed and reservations have been cleared.\n')
 
 # If NI-SCOPE API or Session API throws an exception, print the error message.
-except grpc.RpcError as e:
```

Comment:
We do need to add `finally` block here to close session created on line 92 above.

Reply:
@doshirohan Finally Block is not needed here because on line 160 ResetServer API is called which basically closes any session that the server is associated with.
I even included it and tested but got the error "Session Handle is not valid" while executing the finally block.